fleet/backend/pktvisor/pktvisor.go

```diff
@@ -62,6 +76,38 @@ func (p pktvisorBackend) Inputs() (_ types.Metadata, err error) {
 	return handlers, nil
 }
 
+func RetrieveAgentMetadataByOwner(ctx context.Context, ownerID string, db *sqlx.DB) ([]types.Metadata, error) {
```

Comment:
Why does this have its own and it's not using the one from repo?

Reply:
Because it creates a cycle reference, because we import pktvisor on service... and if we use the repo, than we will import agent_service, that implements the service interface, on pkvisor...

Reply:
Ok I don't want to introduce direct database access here though, let's see how else we can solve it